templates/battle/action_prompt.php

```diff
@@ -475,10 +475,10 @@ function handleKeyUp(event) {
                                             <?php
                                             if ($effect->effect != "none") {
                                                 echo in_array($effect->effect, ["substitution", "counter", "piercing", "recoil"])
-                                                    ? System::unSlug($effect->effect) . " (" . round($effect->effect_amount, 0) . "%)"
+                                                    ? System::unSlug($effect->effect) . " (" . round($effect->display_effect_amount, 0) . "%)"
```

Comment:
Not blocking, but could consider making this a method for easier maintenance.
`$effect->getDisplay()`

Reply:
SGTM